docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+## History
+
+The Windows Console has provided a broad API surface for client command-line applications to manipulate both the output display buffer and the user input buffer. However, other non-Windows platforms have never afforded this specific API-driven approach to their command-line environments, choosing instead to use virtual terminal sequences embedded within the standard input and standard output streams. For a time, Microsoft supported this behavior too in early editions editions of DOS and Windows through a driver called ANSI.SYS.
+
+By contrast, all other platforms derive their command-line environment operations from various dialects of virtual terminal sequences. These sequences are rooted in an [ECMA Standard](https://www.ecma-international.org/publications/standards/Ecma-048.htm) and series of extensions by many vendors tracing back to Digital Equipment Corporation and Tektronix terminals through more modern and common software terminals like [xterm](https://invisible-island.net/xterm/). Many extensions exist within the virtual terminal sequence domain and some sequences are more widely supported than others, but it is safe to say that the world has standardized on this as the command language for command-line experiences with a well-known subset being supported by virtually every terminal and command-line client application.
+
+## Comparisons
+
+### Cross-Platform Support
+
+As every platform in the world, except Windows, natively spoke virtual terminal sequences, both terminal applications and command-line utilities have been easily portable between versions and variations of those operating systems. By contrast, Windows APIs are only supported on Windows. An extensive adapter or translation library has to be written between Windows and VT or vice versa when attempting to port command-line utilities from one platform or another.
+
+### Remote Access
+
+Virtual Terminal Sequences hold a major advantage for remote access as they require no additional work to transport or perform remote procedure calls over what is required to set up a standard remote command-line connection. Simply connecting an outbound and an inbound transport channel (or a single bidirectional channel) over a pipe, socket, file, serial port, or any other device is sufficient to completely carry all information necessary required for an application speaking these sequences to a remote host. On the contrary, the Windows Console APIs have only been accessible on the local machine and all efforts to remote them would require building an entire remote calling and transport interface layer beyond just a simple channel.

```

Comment:
`necessary required` - choose one

Reply:
(consider hard wrappign document at 120 columns - makes review easier because you can target specific lines ;P)

Reply:
I don't know how to do that.